### DISCUSSION TITLE
can׳t subscribe to $SYS/# or #

### DISCUSSION THREAD

**Discussion by roysG** [emqx/emqx#6375] [Q&A]
I try to subscribe to $SYS, but i get error in authrization.
I tried with allowes anonymous login, (no required for username and password) 
and also tried to set user and pass.

in both case i get this error message (attached image
<img width="1350" alt="Screen Shot 2021-12-04 at 12 16 55" src="https://user-images.githubusercontent.com/7311802/144706942-18b8503b-f035-46eb-b303-424fc0c49055.png">
)

**Comment by k32** [comment]
As the error message suggests, the client lacks the permissions to subscribe to the topic. 

> I tried with allowes anonymous login, (no required for username and password)
> and also tried to set user and pass.

This error message is related to authorization rather than authentication. Please check acl.conf to see which rules are set for the SYS topic. By default, the following users are allowed to subscribe to the topic:

```
{allow, {username, "^dashboard?"}, subscribe, ["$SYS/#"]}.

{allow, {ipaddr, "127.0.0.1"}, all, ["$SYS/#", "#"]}.
```

**Reply by roysG** [reply]
Yes it works, thanks!
